plugins/awos/scripts/status-board.py

```
@@ -0,0 +1,241 @@
+        ):
+            in_slice = True
+            continue
+        if re.match(r"\s*- \[[ xX]\]", ln) and ("[Agent:" in ln or in_slice):
+            total += 1
+    return total
+
+
+def _git_epoch(args):
+    """First epoch-looking timestamp from `git log -1 --format=%ct <args>`, or None."""
+    try:
+        out = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)